app/main.py

```diff
@@ -1,8 +1,24 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
```

Comment:
The `people` dictionary is a class variable, which means it is shared across all instances of the `Person` class. This can lead to unexpected behavior when managing multiple `Person` instances. Consider moving the `people` dictionary to an instance variable by defining it within the `__init__` method.